cli/file_fetcher.rs

```diff
@@ -395,10 +398,18 @@ impl FileFetcher {
           specifier
         ),
       ))
-      .boxed_local();
+      .boxed();
     }
 
-    info!("{} {}", colors::green("Download"), specifier);
+    info!(
```

Comment:
I'm really not happy with this, but it was the only thing I could think of to easily fix this.  The debug console in VSCode does not support colours, and this is buried deep in file fetcher and isn't very easy to conditionally suppress... it is useful to see files being downloaded in the debug output for the lsp, so I don't want to totally turn it off... any bright ideas would be appreciated.

Reply:
VSCode seems to solve do this by passing a `*ProgressReporter` in such cases.

https://github.com/microsoft/vscode-languageserver-node/blob/17cbbdc9cc278b5bc697a9461b4eb0d4bab5c49f/server/src/common/progress.ts#L102

Reply:
@kitsonk Can't we just start lsp with NO_COLOR set from VS Code?

Reply:
> Can't we just start lsp with NO_COLOR set from VS Code?

Yes, I guess that would work for now.  It is still challenging having logging of info stuff buried so deeply in the cli at times... we really should be figuring out ways to pull this up to the top, so the consumer can determine what is appropriate or not.

I would add it to my cache command PR, but someone already merged it. 🤷